tasks/main.yml

```diff
@@ -120,13 +120,24 @@
     role_namespace: "{{ osbs_namespace }}"
     serviceaccounts: "{{ osbs_service_accounts }}"
 
+  register: yaml_rolebindings
+  when: osbs_is_admin
+  tags:
+  - oc
+
+- name: copy pruner role binding
+  template:
+    src: "openshift-rolebinding.{{ item.yaml_version | default('v2') }}.yml.j2"
+    dest: "{{ osbs_openshift_home }}/{{ inventory_hostname }}-{{ osbs_namespace }}-rolebinding-{{ item.name }}.yml"
+  with_items:
+
   - name: osbs-pruner-serviceaccounts
     role: system:image-pruner
     type: ClusterRoleBinding
     serviceaccounts: ["{{ osbs_serviceaccount_pruner }}"]
 
   register: yaml_rolebindings
-  when: osbs_is_admin
+  when: osbs_is_admin and osbs_serviceaccount_pruner is defined and osbs_serviceaccount_pruner
```

Comment:
won't this overwrite previous yaml_rolebingings?

Reply:
Right, fixed